manifests/n/NirSoft/AdvancedRun/1.51/NirSoft.AdvancedRun.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NirSoft.AdvancedRun
+PackageVersion: 1.51
+InstallerLocale: en-US
+InstallerType: zip
+Installers:
+- Architecture: x64
```

Comment:
Please also add the x86 version - https://www.nirsoft.net/utils/advancedrun.zip

Reply:
added :)